tests/unittests/CoreGraphics.Drawing/CGContextDrawing_FillTests.cpp

```diff
@@ -0,0 +1,105 @@
+//******************************************************************************
```

Comment:
I'll take this on as I wrote this code. I'm hoping to reduce merge conflicts from develop in the meantime.